mautic/mautic
# Fixed chosen UI for form fields and converted actions to chosen as well

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This simply updates the UI for the dropdown to Chosen conversion for form fields and also converts actions to use a chosen so it looks the same

#### Steps to test this PR:
1.  Add and edit a standalone and campaign form
2. Add fields and actions and ensure they work
3. Ensure the appropriate actions show for campaign vs standalone forms
